documentation/project-docs/developer-guide.md

```diff
@@ -101,6 +101,15 @@ This should output `.../artifacts/tmp/Debug/dotnet/dotnet`.
 
 You can now run `dotnet` commands to test changes.
 
+## Run tests in command line
+
+```shell
+build.cmd # to have a full build first
+.\artifacts\sdk-build-env.bat
+cd src\Tests\YOURTEST.Tests # cd to the test folder that contains the test csproj file
+dotnet test --filter "FullyQualifiedName~TESTNAME" # run individual test
```

Comment:
I typically do something like `dotnet run -method NameSpace.Class.TestMethod` instead.  But I'm fine with doc'ing it this way.